lib/src/ast/sass/statement/forward_rule.dart

```diff
@@ -74,6 +75,9 @@ class ForwardRule implements Statement {
 
   final FileSpan span;
 
+  FileSpan get urlSpan =>
+      span.subspan(RegExp(r'@forward\s+').matchAsPrefix(span.text)!.end);
```

Comment:
Similarly, I'd write a `withoutInitialAtRule()` span helper that you can use here and below.

Also, don't forget that a `@forward` can have `show`/`hide`/`with` clauses that will need to be excluded here.